Remotion/Web/UnitTests/Core/UI/Controls/CommandTests/HrefCommandTest.cs

```diff
@@ -120,5 +121,31 @@ public void Render_WithAccessDenied ()
       Assert.AreEqual(HtmlTextWriterTag.A, _testHelper.HtmlWriter.Tag, "Wrong Tag");
       Assert.AreEqual(0, _testHelper.HtmlWriter.Attributes.Count, "Has Attributes");
     }
+
+    [Test]
+    public void GetCommandInfo_WithEmptyHrefAndParameters_DoesNotResolveClientUrl ()
```

Comment:
I think we should update the comment message. true, we found it via the resoruce prefix, but the underlying issue is not related. Rather, something along the lines of "href with only parameters and no location is not resolved correctly"